### PR TITLE
declare tmpnb deprecated, point to new resources

### PR DESCRIPTION
**Pull request by rgbkrk**
Since we're getting a flurry of new issues and I think we've all acknowledged JupyterHub + Binder as the ~~one true way~~ way to customize your own setup, I think it's time we declare it as such in the repo. Open to wordsmithing here.

/cc @willingc @yuvipanda @minrk

### CODE REVIEW

**Comment by yuvipanda** [comment]
Long time dream of mine, so +1 :)

File: README.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 Launches "temporary" Jupyter notebook servers.
 
+**WARNING: tmpnb is no longer actively maintained.**
+
+We recommend you switch to using JupyterHub.
+
+### Configuration option 1:
+
+* JupyterHub with [tmpauthenticator](https://github.com/jupyterhub/tmpauthenticator)
```
Comment (willingc) [inline README.md]: Let's go ahead and add a link to JupyterHub as well.

**Review by willingc — APPROVED** [review]
Thanks @rgbkrk. I was thinking the same thing this morning. One suggestion to add a link to JupyterHub since tmpnb is currently under the Jupyter org. Let's leave for @minrk to comment and merge.

File: README.md (hunk 1 above)
Comment (rgbkrk) [reply README.md]: Done!

**Review by willingc — APPROVED** [review]

**Comment by yuvipanda** [comment]
Do we have a quick 'howto' anywhere on setting up a tmpnb equivalent with jupyterhub? Would be nice to have a good answer to https://github.com/jupyterhub/tmpauthenticator/issues/10

**Comment by rgbkrk** [comment]
I don't think we do, which is why I kind of waved my hands at "hey here are some tools to solve what you need to make something tmpnb-like".

**Comment by rgbkrk** [comment]
Merging, let's move on forward.

**Comment by willingc** [comment]
I've gone ahead and updated the repo description and site:
screen shot 2018-01-24 at 7 10 50 am

**Comment by rgbkrk** [comment]
Thanks! I'll also mark it as archived.

**Pull request merged**